dist/images/env-check.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set +e
+checkresult=/var/log/kube-ovn/env-check.log
+echo "Start environment check" > $checkresult
+echo "1) check cni configuration" >> $checkresult
+if [ ! -e "/etc/cni/net.d" ]; then
+  echo "Directory /etc/cni/net.d does not exist, please check kube-ovn-cni pod status" >> $checkresult
+fi
+for file in $(ls "/etc/cni/net.d")
+do
+  if [[ ! $file =~ "kube-ovn.conflist" ]]; then
+    echo "Check files in /etc/cni/net.d, make sure if the config file $file should be deleted" >> $checkresult
+  fi
+done
+
+echo "2) check system ipv4 config" >> $checkresult
+probe_mtu=`cat /proc/sys/net/ipv4/tcp_mtu_probing`
+if [ $probe_mtu == 0 ]; then
+  echo "The 'tcp_mtu_probing' config may affect traffic, make sure if /proc/sys/net/ipv4/tcp_mtu_probing should be set to 1" >> $checkresult
+fi
+recycle=`cat /proc/sys/net/ipv4/tcp_tw_recycle`
+if [ $recycle == 1 ]; then
+  echo "The 'tcp_tw_recycle' config affects nodeport service, make sure change /proc/sys/net/ipv4/tcp_tw_recycle to 0" >> $checkresult
+fi
+
+echo "3) check checksum value" >> $checkresult
+netstat -s > /dev/null
```

Comment:
we can use `which netstat` to check whether the command exists.

Reply:
modified